bug/snapshot.go

```diff
@@ -12,16 +12,16 @@ import (
 type Snapshot struct {
 	id entity.Id
 
-	Status       Status
-	Title        string
-	Comments     []Comment
-	Labels       []Label
-	Author       identity.Interface
-	Actors       []identity.Interface
-	Participants []identity.Interface
+	status       Status
+	title        string
+	comments     []Comment
+	labels       []Label
+	author       identity.Interface
```

Comment:
An author shouldn't be stored as an in memory reference. Instead reference an author by ID.
This should decouple both aggregates (snapshot and identity) even further.

Reply:
Done within https://github.com/GlancingMind/git-bug/pull/62/commits/eae0e377adc9e4162239b3bdbbba46bc70592cae